src/migrations/20190829012904-create-multi-request.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('MultiRequests', {
```

Comment:
It does not make sense to have two tables as `request` table(request and multi requests), this means our application is not scalable. We are going to have one table which is `request` table then the destination field will be an array to accept multiple destinations.

Reply:
Noted